blog/2023-03-10-conda-survey.md

```diff
@@ -0,0 +1,194 @@
+
+# What needs improvement?
+
+The survey asked respondents what changes should be prioritized going forward.
+
+[![Desired conda changes](/img/2023-03-10-conda-survey/desired-conda-changes.png)](https://docs.google.com/spreadsheets/d/1rpWN6atSEsALvStPtAVIf_WMTxXXKaDKYpKRsvMpbOo/edit#gid=1229097249)
+
+## Speed
+
+The need for speed was the most prominent theme:
+
+* 71% prioritized speedups
+* 60% use mamba, which is known for being very fast compared to the standard conda solver.
+
+Some individual responses:
+
+* "just make libmamba the default solver already!!!!"
+* "Nearly 100% use Mamba nowadays due to speed, only fallback to Conda when something is going wrong"
+* "*Dear Goddess make the solver faster out of the box.*"
+
+This survey was taken in November and December 2022, which we feel was an inflection point in conda's performance profile.  The [22.11.0 conda release], which came out towards the end of the survey window, implemented parallel package download and extraction, and dropped the `experimental` tag from the `conda-libmamba-solver` (the solver was added in the March 2022 release). At the time of the survey, [only 19% of responsdents were using conda-libmamba-solver](https://docs.google.com/spreadsheets/d/1rpWN6atSEsALvStPtAVIf_WMTxXXKaDKYpKRsvMpbOo/edit#gid=1650032116).
+
+These changes are representative of a multi-year, and continuing effort to improve the speed of conda.  Our next survey will tell us if conda is succeeding at speeding things along...
+
+## Better error messages
+
+Close behind speed was *more helpful error messages* with 65% of respondents.
+
+Some individual feedback
+
+* "when it works, it's fantastic. when it doesn't, it's really painful to diagnose dependency resolution issues and discover what's happening, resulting in ad hoc tools like parsing json into networkx or something for exploration."
+* "Explorability of dependency resolution conflicts"
+
+The conda-libmamba-solver error messages will improve shortly [as the latest version of the solver gets integrated into conda](https://github.com/conda/conda-libmamba-solver/issues/102).
+
+## Better interoperability
+
+*Improved interoperability with other package managers* was prioritized by 62% of respondents. This message is reinforced by how frequently people use more than one package manager.
+
+[![Which additional package managers do your use, top](/img/2023-03-10-conda-survey/which-additional-package-managers-do-you-use-top.png)](https://docs.google.com/spreadsheets/d/1rpWN6atSEsALvStPtAVIf_WMTxXXKaDKYpKRsvMpbOo/edit#gid=1102296811)
+
+79% of conda survey respondents use other package managers in addition to conda.  75% of all respondents also use pip, and 55% use mamba.
+
+The PSF survey asked "What Packaging tools do you use?"  They blended together package and environment management (as does conda itself):
```

Comment:
```suggestion
The PSF survey asked "What Packaging tools do you use?".  They blended together package and environment management (as does conda itself):
```

Reply:
Hmmm.  I don't think we need that period.  I think the question mark ends the sentence.